go/interpreter/lfvm/instructions_test.go

```diff
@@ -1173,3 +1173,71 @@ func TestOpEndWithResult_ReportOverflow(t *testing.T) {
 		t.Fatalf("should have produced overflow error, instead got: %v", err)
 	}
 }
+
+func TestInstructions_JumpInstructionsCheckJUMPDEST(t *testing.T) {
+	tests := map[OpCode]struct {
+		implementation func(*context) error
+		stack          []uint64
+	}{
+		JUMP: {
+			implementation: opJump,
+			stack:          []uint64{1},
+		},
+		JUMPI: {
+			implementation: opJumpi,
+			stack:          []uint64{1, 1},
+		},
+		SWAP2_SWAP1_POP_JUMP: {
+			implementation: opSwap2_Swap1_Pop_Jump,
+			stack:          []uint64{1, 1, 1},
+		},
+		POP_JUMP: {
+			implementation: opPop_Jump,
+			stack:          []uint64{1, 1},
+		},
+		PUSH2_JUMP: {
+			implementation: opPush2_Jump,
+			stack:          []uint64{1},
+		},
+		PUSH2_JUMPI: {
+			implementation: opPush2_Jumpi,
+			stack:          []uint64{1},
+		},
+		ISZERO_PUSH2_JUMPI: {
+			implementation: opIsZero_Push2_Jumpi,
+			stack:          []uint64{1, 1, 0},
+		},
+	}
+
+	// test that all jump instructions are tested
+	for _, op := range allOpCodesWhere(isJump) {
+		if _, ok := tests[op]; !ok {
+			t.Fatalf("missing test for jump instruction %v", op)
+		}
+	}
+
+	for op, test := range tests {
+		t.Run(op.String(), func(t *testing.T) {
+
+			ctxt := context{
+				params: tosca.Parameters{
+					BlockParameters: tosca.BlockParameters{
+						Revision: tosca.R13_Cancun,
+					},
+				},
+				stack:  NewStack(),
+				memory: NewMemory(),
+				gas:    1 << 32,
```

Comment:
unnecessary specifications.

Reply:
gas is needed, but memory not.

Reply:
sorry, yes, gas does not need to be so big, but it's fine as is